spec/actionizer/inputs_spec.rb

```diff
@@ -51,8 +51,10 @@ def call; end
           def call; end
         end
       end
-      it 'raises an error' do
-        expect { failing_class.call(bar: 'oops') }.to raise_error(ArgumentError)
+      it 'fails gracefully and sets the error message' do
+        result = failing_class.call(bar: 'oops')
+        expect(result).to be_failure
+        expect(result.error).not_to be_nil
```

Comment:
I thought about it, and it would be useful, although it would also make the tests more brittle. I guess in this context the project won't grow like a web app might, so it would probably be safe to do so.